mppe/main.go

```diff
@@ -14,35 +19,61 @@ func main() {
 	year := flag.Int("ano", 0, "Ano a ser analisado")
 	flag.Parse()
 	if *month == 0 || *year == 0 {
-		logError("month or year not provided. Please provide those to continue. --mes={} --ano={}\n")
-		os.Exit(1)
+		e := status.NewError(status.InvalidParameters, errors.New("month or year not provided. Please provide those to continue. --mes={} --ano={}"))
+		status.ExitFromError(e)
 	}
 	if *year < 2011 {
-		logError("years before 2011 are not supported yet :(")
-		os.Exit(1)
+		e := status.NewError(status.InvalidParameters, errors.New("years before 2011 are not supported yet :("))
+		status.ExitFromError(e)
 	}
 	if *month < 1 || *month > 12 || *month <= 0 {
-		logError("invalid month value. Give values between 1 and 12")
-		os.Exit(1)
+		e := status.NewError(status.InvalidParameters, errors.New("invalid month value. Give values between 1 and 12"))
+		status.ExitFromError(e)
 	}
 	if *year <= 0 {
-		logError("invalid year value. Give years from and above 2011")
-		os.Exit(1)
+		e := status.NewError(status.InvalidParameters, errors.New("invalid year value. Give years from and above 2011"))
+		status.ExitFromError(e)
 	}
 	outputFolder := os.Getenv("OUTPUT_FOLDER")
 	if outputFolder == "" {
 		outputFolder = "./output"
 	}
 	if err := os.Mkdir(outputFolder, os.ModePerm); err != nil && !os.IsExist(err) {
-		logError("error creating output folder(%s): %q", outputFolder, err)
-		os.Exit(1)
+		e := status.NewError(status.SystemError, fmt.Errorf("error creating output folder(%s): %q", outputFolder, err))
+		status.ExitFromError(e)
 	}
 	paths, err := Crawl(outputFolder, *month, *year, baseURL)
 	if err != nil {
-		logError("error on crawling: ", err.Error())
-		os.Exit(1)
+		status.ExitFromError(err)
+	}
+	employees, err := Parse(paths)
+	if err != nil {
+		status.ExitFromError(err)
+	}
+	crawlingResult := newCrawlingResult(employees, paths, *month, *year)
```

Comment:
Nome muito grande? :)

```suggestion
	cr := newCrawlingResult(employees, paths, *month, *year)
```